zebra-rpc/src/methods.rs

```diff
@@ -171,6 +172,10 @@ pub trait Rpc {
     #[rpc(name = "getbestblockhash")]
     fn get_best_block_hash(&self) -> Result<GetBlockHash>;
 
+    /// Returns the height and hash of the current best blockchain tip block, as a [`GetBlockHeightAndHash`] JSON struct.
```

Comment:
In order for the openapi spec utility script to work we need to add some more documentation to new methods.

I think this will be enough here:

```suggestion
    /// Returns the height and hash of the current best blockchain tip block, as a [`GetBlockHeightAndHash`] JSON struct.
    ///
    /// zcashd reference: none
    /// method: post
    /// tags: blockchain
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/8596/commits/15feed173daced582340fdcdb4d4bb8cc9593a39.